### PR TITLE
Update doc url

### PR DESCRIPTION
**Pull request by VincentLanglet**
Link need to be updated when merging into 4.x

### CODE REVIEW

File: docs/getting_started/installation.rst [hunk 1]
```diff
@@ -159,6 +159,6 @@ provided admin functionality for the admin bundle yet. Fortunately, you'll
 learn how to do this in the :doc:`next chapter <creating_an_admin>`.
 
 .. _`installation chapter`: https://getcomposer.org/doc/00-intro.md
-.. _SonataDoctrineORMAdminBundle: https://sonata-project.org/bundles/doctrine-orm-admin/master/doc/index.html
-.. _SonataDoctrineMongoDBAdminBundle: https://sonata-project.org/bundles/mongo-admin/master/doc/index.html
-.. _SonataDoctrinePhpcrAdminBundle: https://sonata-project.org/bundles/doctrine-phpcr-admin/master/doc/index.html
+.. _SonataDoctrineORMAdminBundle: https://docs.sonata-project.org/projects/SonataDoctrineORMAdminBundle/en/3.x
```
Comment (jordisala1991) [inline docs/getting_started/installation.rst:162]: this might need a last /, otherwise it gets redirected.

Comment (jordisala1991) [reply docs/getting_started/installation.rst:162]: Same for other links.

Comment (VincentLanglet) [reply docs/getting_started/installation.rst]: better ?

File: docs/reference/security.rst [hunk 2]
```diff
@@ -788,6 +788,6 @@ return an iterable collection of roles::
     }
 
 
-.. _`SonataUserBundle's documentation area`: https://sonata-project.org/bundles/user/master/doc/reference/installation.html
+.. _`SonataUserBundle's documentation area`: https://docs.sonata-project.org/projects/SonataUserBundle/en/4.x/reference/installation
```
Comment (jordisala1991) [inline docs/reference/security.rst:791]: missing /

**Review by jordisala1991 — CHANGES_REQUESTED** [review]

File: docs/reference/security.rst (hunk 2 above)
Comment (VincentLanglet) [reply docs/reference/security.rst]: done

**Review by jordisala1991 — APPROVED** [review]

**Pull request merged**